Solutions/MSTelemetrySDK.sln

```diff
@@ -130,8 +130,6 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "SampleCppLogManagers", "..\
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "SampleCppMini", "..\examples\cpp\SampleCppMini\SampleCppMini.vcxproj", "{86AC752C-5687-4377-841E-943D9BEEF361}"
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "SampleCppUTC", "..\lib\modules\examples\cpp\SampleCppUTC\SampleCppUTC.vcxproj", "{C947B185-2B00-4073-A19D-483ED5C7EDC2}"

```

Comment:
Why is this change needed?

Reply:
Since your `powershell` script already does:

```powershell
& cmd /c "msbuild $solution /target:$targetStr /p:BuildProjectReferences=true /maxcpucount:$cpuCount /p:Configuration=$actualConfig /p:Platform=$actualArch $actualCustomProps"
```

i.e. selective build of only the libraries you need via `/target:$targetStr` option, it seems to me that the changes made to Solution file here have no effect. You should be able to safely revert the .sln change. These are needed for customers who have extra modules under `lib/modules/` path.

Reply:
I was getting errors when I tried to build the solution file (couldn't find the project). If SampleCppUTC is needed by other customers, I'll revert that part of the change and just ensure it doesn't get scheduled for the arm64 lib build.

Reply:
Sounds good.

Reply:
You can also make sure that you can recursively clone with "private" module, that way your snapshot will include the UTC sample app too. It'd be best if we retain it in the solution.